core/src/main/scala/org/bykn/bosatsu/PackageMap.scala

```diff
@@ -52,7 +52,24 @@ object PackageMap {
   type MapF2[A, B] = MapF3[A, A, B]
   type ParsedImp = PackageMap[PackageName, Unit, Unit, (Statement, ImportMap[PackageName, Unit])]
   type Resolved = MapF2[Unit, (Statement, ImportMap[PackageName, Unit])]
-  type Inferred = PackageMap[Package.Interface, NonEmptyList[Referant[Variance]], Referant[Variance], Program[TypeEnv[Variance], TypedExpr[Declaration], Statement]]
+  type Typed[T] = PackageMap[
+    Package.Interface,
+    NonEmptyList[Referant[Variance]],
+    Referant[Variance],
+    Program[
+      TypeEnv[Variance],
+      TypedExpr[T],
+      Statement
+    ]
+  ]
+  
+  type Inferred = Typed[Declaration]
+
+  type Normalized = Typed[(Declaration, Normalization.NormalExpressionTag)]
+
+  def normalizePackages[A, B, C, D](map: Inferred): Normalized = {
+    PackageMap.empty
```

Comment:
this looks suspect... Can we add a comment or leave it as `???`

Reply:
oops, gone